src/workloads/execution/MultiPlanningReadsALotOfData.yml

```diff
@@ -0,0 +1,114 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: >
+  Create collection with multiple indexes and run queries for which multi planner needs to read a lot
+  of data in order to pick the best plan.
+
+Keywords:
+  - indexes
+
+Actors:
+  - Name: InitCollection
+    Type: RunCommand
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [0]
+        NopInPhasesUpTo: &MaxPhase 3
+        PhaseConfig:
+          Repeat: 1
+          Database: &Database test
+          Operations:
+            - OperationName: RunCommand
+              OperationCommand:
+                createIndexes: &Collection Collection0
+                indexes:
+                  - key:
+                      flag_a: 1
+                      flag_b: 1
+                    name: flags
+                  - key:
+                      int_a: 1
+                    name: int
+
+  - Name: DisablePlanCache
+    Type: AdminCommand
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [0]
+        NopInPhasesUpTo: *MaxPhase
+        PhaseConfig:
+          Repeat: 1
+          Database: admin
+          Operations:
+            - OperationName: AdminCommand
+              OperationCommand:
+                setParameter: 1
+                internalQueryDisablePlanCache: true
+
+  - Name: Insert
+    Type: Loader
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [1]
+        NopInPhasesUpTo: *MaxPhase
+        PhaseConfig:
+          Repeat: 1
+          Database: *Database
+          Threads: 1
+          CollectionCount: 1
+          DocumentCount: 2e6
+          BatchSize: 1000
+          Document:
+            flag_a: &randomBit { ^RandomInt: { min: 0, max: 1 } }
+            flag_b: *randomBit
+            int_a: { ^RandomInt: { min: 1, max: 10 } }
+            int_b: { ^Inc: {} }
+
+  - Name: FindWithMultiplanning
```

Comment:
Should we separate the find and agg part as different actors with different names: (`FindMultiplanNoIndexOnSelectivePredicate` or something similar)? Do we want combined metrics data for both? Having a separate actor will give more fine grained visibility.

Reply:
We don't need another actor, we can distinguish per operation name: 
```
QueryWithMultiplanning.find.ftdc
QueryWithMultiplanning.aggregate.ftdc
```